deployer/src/infra/nats.rs

```diff
@@ -0,0 +1,40 @@
+use crate::infra::{async_trait, Builder, ComponentAction, ComposeTest, Error, Nats, StartOptions};
+use composer::ContainerSpec;
+use events_api::mbus_nats::message_bus_init;
+use std::ffi::OsString;
+
+/*
+Steps to check the events.
+
+-- Create nats box.
+docker run --network cluster --rm -it natsio/nats-box
+-- Check the number of events in the stream.
+nats -s nats://nats:4222 str ls
+*/
+
+#[async_trait]
+impl ComponentAction for Nats {
+    fn configure(&self, options: &StartOptions, cfg: Builder) -> Result<Builder, Error> {
+        Ok(if !options.eventing {
+            cfg
+        } else {
+            cfg.add_container_spec(
+                ContainerSpec::from_image("nats", "nats:latest")
```

Comment:
If we add the nats binary to the shell and use the binary instead of the container, will make it easier for debug?
Alternatively we should at least pin nats to a particular version rather than latest.